docker-extension/src/main/java/org/ballerinax/docker/DockerPlugin.java

```diff
@@ -63,8 +64,10 @@ public void init(DiagnosticLog diagnosticLog) {
 
     @Override
     public void process(PackageNode packageNode) {
-        String pkgID = ((BLangPackage) packageNode).packageID.toString();
-        DockerContext.getInstance().addDataHolder(pkgID);
+        if (!(packageNode instanceof BLangTestablePackage)) {
```

Comment:
We can make this code readable by skipping if the package is a testable package.